### PR TITLE
Enable QTI backend to use ndk 25

### PR DESCRIPTION
**Pull request by mohitmundhragithub**

### CODE REVIEW

File: flutter/android/app/build.gradle [hunk 1]
```diff
@@ -22,10 +22,6 @@ if (flutterVersionName == null) {
 }
 
 apply plugin: 'com.android.application'
-// START: FlutterFire Configuration
```
Comment (mohitmundhragithub) [inline flutter/android/app/build.gradle:25]: @anhappdev I was getting some gradle errors when building with firebase.
Can you please help with the correct command to be used. mostly the build id that i used is wrong.

File: flutter/android/docker/Dockerfile [hunk 2]
```diff
@@ -75,3 +75,4 @@ RUN chmod -R 754 $HOME
 RUN mkdir -p /mnt/cache && chmod 777 /mnt/cache
 RUN mkdir -p /mnt/project/flutter/build && chmod 777 /mnt/project/flutter/build
 RUN mkdir -p $HOME/.cache/bazelisk && chmod 777 $HOME/.cache/bazelisk
+ENV FIREBASE_PROJECT_ID=0
```
Comment (mohitmundhragithub) [inline flutter/android/docker/Dockerfile:78]: @anhappdev this is the build id i am talking about in the previous comment.

Comment (anhappdev) [reply flutter/android/docker/Dockerfile:78]: Did you set up Firebase env vars as described here?
https://github.com/mlcommons/mobile_app_open/blob/master/docs/environment-setup/env-setup-firebase.md

File: flutter/android/app/build.gradle (hunk 1 above)
Comment (anhappdev) [reply flutter/android/app/build.gradle:25]: Did you set up Firebase env vars as described here?
https://github.com/mlcommons/mobile_app_open/blob/master/docs/environment-setup/env-setup-firebase.md

File: flutter/android/docker/Dockerfile (hunk 2 above)
Comment (anhappdev) [reply flutter/android/docker/Dockerfile:78]: You don’t need to add this line.

File: flutter/windows/flutter/generated_plugin_registrant.cc
Comment (anhappdev) [inline flutter/windows/flutter/generated_plugin_registrant.cc:1]: Firebase don’t work with Windows yet. So you need to manually ignore (do not commit) the changes addes to 2 files:
flutter/windows/flutter/generated_plugin_registrant.cc
flutter/windows/flutter/generated_plugins.cmake

Comment (anhappdev) [reply flutter/windows/flutter/generated_plugin_registrant.cc:1]: Or it would be great if you can help make the Windows build work with Firebase.

Comment (mohitmundhragithub) [reply flutter/windows/flutter/generated_plugin_registrant.cc:1]: I will work on this in a separate PR. for now, will remove these changes.

**Review by anhappdev — APPROVED** [review]

**Pull request merged**